src/main/java/ua/danit/final_project/services/WorkCommentServiceImpl.java

```diff
@@ -66,10 +59,13 @@ public ShiftComment updateComment(ShiftComment shiftComment) {
 
   @Override
   public List<ShiftComment> getCommentsOfLastWorkShifts(Long workShiftId) {
-    Integer id = shiftCommentRepository.getMaxId();
-    Long maxId = Long.parseLong("" + (id - 2));
+    Timestamp date = new Timestamp(System.currentTimeMillis());
+    DateTime searchDate = new DateTime(date).withTimeAtStartOfDay();
+    Date from = searchDate.toDate();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "from". ([squid:S1854](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=88&issues=AWWVbxDV7khxiKmGg8xQ&open=AWWVbxDV7khxiKmGg8xQ)

[ISSUE_KEY]: AWWVbxDV7khxiKmGg8xQ